data/json/items/chemicals_and_resources.json

```diff
@@ -951,5 +951,19 @@
     "weight" : 1,
     "ammo_type" : "components",
     "count" : 100
-   }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "ether",
+    "category": "chems",
+    "price": 135000,
```

Comment:
Price is too high:

![image](https://user-images.githubusercontent.com/16213433/49691582-0b976d80-fb57-11e8-978e-d9d41f7572eb.png)

Reply:
reduced it to 10